content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+
+```bash
+kubeadm token create
+```
+
+Keluaran akan tampil seperti berikut:
+
+```console
+5didvk.d09sbcov8ph2amjw
+```
+
+Jika kamu tidak memiliki nilai `--discovery-token-ca-cert-hash`, kamu bisa mendapatkannya dengan menjalankan perintah berantai berikut pada simpul _control-plane_:
+
+```bash
+openssl x509 -pubkey -in /etc/kubernetes/pki/ca.crt | openssl rsa -pubin -outform der 2>/dev/null | \
+   openssl dgst -sha256 -hex | sed 's/^.* //'
+```
+
+Keluaran yang diberikan kurang lebih akan ditampilkan sebagai berikut:
+
+```console
+8cb2de97839780a412b93877f8507ad6c94f73add17d5d7058e91741c9d5ec78
+```
+
+{{< note >}}
+Untuk menentukan _tuple_ IPv6 untuk `<control-plane-host>:<control-plane-port>`, alamat IPv6 harus be ditutup dengan kurung siku, sebagai contoh: `[fd00::101]:2073`.
+{{< /note >}}
+
+Keluaran yang diberikan kurang lebih akan ditampilkan sebagai berikut:
+
+```
+[preflight] Running pre-flight checks
+
+... (log output of join workflow) ...
+
+Node join complete:
+* Certificate signing request sent to control-plane and response
+  received.
+* Kubelet informed of new secure connection details.
+
+Run 'kubectl get nodes' on control-plane to see this machine join.
+```
+
+Beberapa saat kemudian, kamu akan melihat simpul tersebut pada keluaran dari `kubectl get nodes` ketika dijalankan pada simpul _control-plane_.
+
+### (Opsional) Mengendalikan klaster dari mesin selain simpul _control-plane_
+
+Untuk membuat kubectl bekerja pada mesin lain (e.g. laptop) agar dapat berbicara dengan
+klaster, kamu harus menyalin berkas kubeconfig administrator dari simpul _control-plane_
+ke _workstation_ seperti berikut:
+
+```bash
+scp root@<control-plane-host>:/etc/kubernetes/admin.conf .
+kubectl --kubeconfig ./admin.conf get nodes
+```
+
+{{< note >}}
+Contoh di atas mengasumsikan akses SSH dinyalakan untuk _root_. Jika tidak berlaku
+demikian, kamu dapat menyalin berkas `admin.conf` untuk dapat diakses oleh pengguna lain
+dan `scp` menggunakan pengguna lain tersebut.
+
+Berkas `admin.conf` memberikan penggunanya privilese _superuser_ terhadap klaster.
+Berkas ini harus digunakan seperlunya. Untuk pengguna biasa, direkomendasikan
+untuk membuat kredensial unik dengan privilese whitelist. Kamu dapat melakukan
+ini dengan perintah `kubeadm alpha kubeconfig user --client-name <CN>`
+. Perintah tersebut akan mencetak berkas KubeConfig ke STDOUT yang harus kamu simpan
+ke dalam sebuah berkas dan mendistribusikannya pada para pengguna. Setelah itu, whitelist
+privilese menggunakan `kubectl create (cluster)rolebinding`.
+{{< /note >}}
+
+### (Opsional) Meneruskan Peladen API ke localhost
+
+Jika kamu ingin terhubung dengan peladen API dari luar klaster kamu dapat menggunakan
+`kubectl proxy`:
+
+```bash
+scp root@<control-plane-host>:/etc/kubernetes/admin.conf .
+kubectl --kubeconfig ./admin.conf proxy
+```
+
+Kini kamu dapat mengakses peladen API secara lokal melalui `http://localhost:8001/api/v1`
```

Comment:
```suggestion
Kini kamu dapat mengakses API Server secara lokal melalui `http://localhost:8001/api/v1`
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc